includes/class-sensei-usage-tracking-data.php

```diff
@@ -34,6 +38,92 @@ public static function get_usage_data() {
 		);
 	}
 
+	/**
+	 * Get the number of courses that have a video set.
+	 *
+	 * @since 1.9.20
+	 *
+	 * @return int Number of courses.
+	 */
+	private static function get_courses_with_video_count() {
+		$query = new WP_Query( array(
```

Comment:
Note that you'll need to add `'posts_per_page' => -1` or it will only return a maximum of 10 posts. Just realized that myself.

Reply:
Also, `'fields' => 'ids',` to return just the post IDs. Otherwise we'll bring back the entire post, which we don't need since we're only getting a count.

Reply:
Alternatively, leaving off `'posts_per_page' => -1` and returning `$query->found_posts` also seems to work.

Reply:
Good catch! I switched to `found_posts` and added `'fields' => 'ids'`